gui/builtinStatsViews/resourcesViewFull.py

```diff
@@ -105,8 +105,13 @@ def populatePanel(self, contentPanel, headerPanel):
         base = sizerResources
         sizer.AddStretchSpacer()
         # Turrets & launcher hardslots display
-        tooltipText = {"turret": "Turret hardpoints", "launcher": "Launcher hardpoints", "drones": "Drones active",
-                       "fighter": "Fighter squadrons active", "calibration": "Calibration"}
+        tooltipText = {
+            "turret"     : _("Turret hardpoints"),
```

Comment:
No padding for vertical dicts please, here and below

Reply:
I added a commit for reformat the code.